pdns/dnsdist-lua.cc

```diff
@@ -1936,6 +1936,10 @@ static void setupLuaConfig(bool client, bool configCheck)
         frontend->d_trustForwardedForHeader = boost::get<bool>((*vars)["trustForwardedForHeader"]);
       }
 
+      if (vars->count("internalPipeBufferSize")) {
+        frontend->d_internalPipeBufferSize = boost::get<int>((*vars)["internalPipeBufferSize"]);
```

Comment:
```suggestion
        frontend->d_internalPipeBufferSize = boost::get<uint32_t>((*vars)["internalPipeBufferSize"]);
```

Reply:
This will not compile because the `uint32_t` type is not present in that variant. I could add it but I would prefer keeping the number of types limited because it quickly becomes confusing :)